openedx/core/djangoapps/signals/signals.py

```diff
@@ -11,3 +11,11 @@
 # TODO: runtime coupling between apps will be reduced if this event is changed to carry a username
 # rather than a User object; however, this will require changes to the milestones and badges APIs
 COURSE_CERT_AWARDED = Signal(providing_args=["user", "course_key", "mode", "status"])
+
+# Signal that indicates that a user has passed a course.
+LEARNER_COURSE_PASSING_GRADE = Signal(
```

Comment:
If the only difference between this signal and `COURSE_GRADE_CHANGED` is to indicate whether the learner is now in a passing state, then this signal is not needed.  The `COURSE_GRADE_CHANGED` already contains the `CourseGrade` object, which already has a `passed` boolean.

However, if this signal is fired only when a learner's grade changes from non-passing to passing, then this signal is worthwhile.  In that case, I would suggest a renaming of the signal, though, to be consistent with the other names in this file.  So something to the effect `COURSE_GRADE_NOW_PASSED`.